vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
-WAR files generated from a core build as dependencies. In a default deployment this is the _entando-de-app_.
+The **entando-core** project is a J2EE application that exposes APIs for the Entando CMS, including the admin console and the portal UI project that performs the server side composition for pages rendered via an Entando application. Note that only the composition is performed server side. JavaScript code is rendered client-side. The **entando-core** is realized via an instance that includes the 
+WAR files generated from a core build as dependencies. In a default deployment this is the **entando-de-app**.
 
 * GitHub: <https://github.com/entando/entando-core/>
 * DockerHub: None (deployed to maven central)
 
 #### Customization
-For users familiar with versions prior to Entando 6, there will be cases where the _entando-core_ is customized. 
-In most cases, these customizations will be delivered via a WAR overlay in the instance project. 
+For users familiar with versions prior to Entando 6, there will be cases where the **entando-core** is customized. 
+Often, these customizations will be delivered via a WAR overlay in the instance project. 
 Using a WAR overlay is a functional approach for users already familiar with the process, but it is highly 
 recommended to extend the platform using microservices for new projects.
 
 ### entando-cms
-The _entando-cms_ project is the _app-builder_ (React JS) side of the Entando WCMS. It is bundled into the 
-_app-builder_ at build time and will be included in the default deployment of the _app-builder_ in almost all cases.
+The **entando-cms** project is the App Builder (React JS) side of the Entando WCMS. It is bundled into the 
+App Builder at build time and will be included in the default deployment of the App Builder in almost all cases.
 * GitHub: <https://github.com/entando/entando-cms/>
 * DockerHub: None (deployed to npm)
 
 #### Customization
-In some cases the _entando-cms_ may be customized if new custom features are added to CMS specific 
-functionality. However, most cases will use the more general _app-builder_ extension points noted above. 
-The _entando-cms_ does not expose any dedicated extension interfaces outside of those already provided by the 
-_app-builder_.
-
-### entando-components
-The entando-components project is a collection of legacy plugins for Entando 5 and earlier. These plugins are deployed as WAR dependencies in an entando-core instance.
-* GitHub: <https://github.com/entando/entando-components/>
-* DockerHub: None (deployed to maven central)
+In some cases the **entando-cms** may be customized if new custom features are added to CMS specific 
+functionality. However, most cases will use the more general App Builder extension points noted above. The **entando-cms** does not expose any dedicated extension interfaces outside of those already provided by the App Builder.
```

Comment:
cases,

new custom features -> new features

CMS-specific

general -> generic

most cases -> most implementations

Reply:
corrected 2 items.
again, it made sense so didn't do a deep dive.

Think 'more general' is better than generic.